src/vaadin-menu-bar-interactions-mixin.html

```diff
@@ -99,10 +109,15 @@
       });
     }
 
+    /** @private */
     _getButtonFromEvent(e) {
       return Array.from(e.composedPath()).filter(el => el.localName === 'vaadin-menu-bar-button')[0];
     }
 
+    /**
+     * @param {!FocusEvent} event
+     * @protected
```

Comment:
Not sure if we want to expose this event listener. Maybe we can mark it as `@private`?

Reply:
Either way is ok to me.